pages/en/lb4/Getting-started.md

```diff
@@ -5,35 +5,202 @@ keywords: LoopBack 4.0, LoopBack 4
+You can also enter values for entries like `git repository` and `keywords`,
+but they are not required.
+
+```
+package name: (lb4-hello-world)
+version: (1.0.0)
+description: Hello World for LoopBack 4
+entry point: (index.js)
+test command:
+git repository:
+keywords:
+author:
+license: (ISC)
+```
+
+The file will then contain the following:
+
+```js
+{
+  "name": "lb4-hello-world",
+  "version": "1.0.0",
+  "description": "Hello World for LoopBack 4",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
+}
+```
+
+**To continue, follow one of these procedures**:
+
+- [JavaScript project](#javascript-project)
+- [TypeScript project](#typescript-project)
+
+## Try a JavaScript project
+
+If you're coding in JavaScript, follow these steps:
+
+1. Create `index.js` file.
+1. Run the project.
+
+### Create `index.js`
+
+Create a file named `index.js`, and copy the following into it:
+
+```js
+const Application = require('@loopback/core').Application;
+
+const app = new Application();
+app.bind('message').to('Hello worldA');
+app.get('message').then(value => {
+  console.log(value);
+});
+```
+
+### Run the project
+
+To run your project, enter this command:
+
+```shell
+node index.js
+```
+
+You should see "Hello world!" written to the console.
+
+## Try a TypeScript project
+
+If you're coding in TypeScript, follow these steps:
+
+1. Create `tsconfig.json` file.
+2. Create `index.ts` file.
+1. Run the project.
+
+### Create `tsconfig.json` file
+
+A `tsconfig.json` file in a directory indicates that it contains a TypeScript project. The `tsconfig.json` file specifies the root files and the TypeScript compiler options. For more information on `tsconfig.json`, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
+
+Create a new `tsconfig.json` file by entering the following command:
+
+```
+$ tsc --init
+```
+
+Now edit this file, because you need to change some of the default values.
+
+{% include important.html content= "You must set the `'compilerOptions.target'` property in  your project's `tsconfig.json` file based on the version of Node you are using.
+" %}
+
+**With Node 6.x**:
+
+```js
+{
+  "compilerOptions": {
+   ...
+    "target": "es6",
+    ...
+  }
+}
+```
+
+**With Node 8.x**:
+
+```js
+{
+  "compilerOptions": {
+    ...
+    "target": "es2017",
+    ...
+  }
+}
+```
+
+### Create `index.ts`
+
+Create a file named `index.ts` and copy the following code into it:
+
+```ts
+import {Application} from '@loopback/core';
+
+const app = new Application();
+app.bind('message').to('Hello world!');
+app.get('message').then(value => {
+  console.log(value);
+});
+```
+
+### Run the project
+
+Then run `index.ts` by entering this command:
+
+```
+ts-node index.ts
```

Comment:
Initially we were using `ts-node` because it promised simpler project setup. Eventually we got frustrated with its shortcomings and ditched it out.

I think that for a quick & short Getting Started Guide it's ok to use ts-node, but we should not be promoting it anywhere else.

As an alternative to a global install of `ts-node`, I am suggesting to define a "start" script that invokes `ts-node index.ts` and move `ts-node` to project dependencies.

```
{
  // ...
  scripts: {
    // ...
    "start": "ts-node index.ts"
  },
  "dependencies": {
    // ...
    "ts-node": "^3.3.0"
  }
}
```

Reply:
I didn't do this, but if you feel strongly that it's preferable, I'll make that change.